packages/musap-react-native/android/src/main/kotlin/com/sphereon/musap/models/SscdType.kt

```diff
@@ -0,0 +1,6 @@
+package com.sphereon.musap.models
+
+enum class SscdType(val value: String) {
+    TEE("TEE"),
```

Comment:
Where is eSim/eUICC and HSM, we would support for the project?

Please create a new ticket for that

Reply:
https://sphereon.atlassian.net/browse/SPRIND-30 BTW